core-client/src/lib.rs

```diff
@@ -284,12 +360,12 @@ pub mod local {
 
 #[cfg(test)]
 mod tests {
-	use futures::prelude::*;
-	use jsonrpc_core_client::local;
+	use super::*;
 	use jsonrpc_core::{IoHandler, Result};
 	use jsonrpc_derive::rpc;
+	use crate::{TypedClient, RpcError, RpcChannel};
 
-	#[rpc]
+	#[rpc(server)]
```

Comment:
Would be good to actually add some tests for the generated client somewhere too. Maybe in a separate crate?